package.json

```diff
@@ -37,6 +37,7 @@
     "@moxy/next-compile-node-modules": "^1.3.2",
     "@moxy/next-intl": "^1.1.4",
     "@moxy/next-webpack-oneof": "^1.0.0",
+    "@moxy/next-with-router-ref": "^1.0.0",
```

Comment:
It seems all added deps are unnecessary, including this one and the redux ones.

Reply:
Yes, some were added while migrating the tests but in the end they were not needed.

Reply:
I think you forgot to handle this as well.